src/components/featured-products-section/featured-products-section.module.scss

```diff
@@ -35,11 +35,6 @@
 }
 
 @media (max-width: $desktop-width) {
-    .root {
-        padding-left: 4%;
-        padding-right: 4%;
-    }
-
```

Comment:
I did this as a part of https://github.com/codux-templates/e-commerce-reclaim/pull/89/files#diff-7298264a04b1a9fc15f092f15eefb5ddd100b85dfb8222bb41f0d59168f30f22